lib/final-api/endpoint/ddtf.rb

```diff
@@ -39,6 +39,15 @@ def registered(app)
             FinalAPI::V1::Http::DDTF_Build.new(build, {}).test_data.to_json
           end
 
+          app.post '/ddtf/tests/:id/retest' do
+            build = Build.find(params[:id])
+            retest_data = FinalAPI::V1::Http::DDTF_Build.new(build, {}).retest_data
+            retest_data['runtimeConfigFields'].reject! do |key, _|
```

Comment:
The delete_if could be a bit more readable, but that is just a question of personal taste and not a merge blocker :)